pkg/client/config.go

```diff
@@ -54,9 +67,10 @@ func (c *Client) AddConfig(config *v1alpha1.K8sGPT) error {
 
 func (c *Client) RemoveConfig(config *v1alpha1.K8sGPT) error {
```

Comment:
Does this function actually work?
RemoveConfigRequest is actually an empty value.

Reply:
RemoveConfig is actually only related to removing the cache configuration, frankly speaking the whole config API that we have created needs better structuring as it has grown organically including different functionality

It _works_  by removing whatever is configured in the k8sgpt's configuration cache stanza..

So neither integrations nor now the customanalyzers will be deleted with removeConfig()

Reply:
Yeah we were just discussing doing a refactor of the config stuff in slack